CHANGELOG.md

```diff
@@ -5,7 +5,8 @@
 - Update `TeleconsultRecord` to sync frequently
 - Add `TeleconsultStatus` to `TeleconsultRecord`
 - Mark strings used in prescription image as un-translatable
-- Use `US` locale when formatting prescription date
+- Use `ENGLISH` locale when formatting prescription date
```

Comment:
Can you explain why are we changing the locale here?

Reply:
We are not actually changing it in this PR, it's just a typo fix in the CHANGELOG from #2021